app/assets/stylesheets/report.scss

```diff
@@ -2,26 +2,39 @@
   background: #fff;
   border-radius: 2px;
   box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
-  display: block;
-  float: left;
-  margin: 1rem;
+  // display: block;
+  // float: left;
+  margin-top: 2rem;
```

Comment:
Properties should be ordered background, border-radius, box-shadow, margin-bottom, margin-top, min-height, padding, transition